### DISCUSSION TITLE
basic tutorial

### DISCUSSION THREAD

**Discussion by atomadam133** [Chia-Network/chia-blockchain#10809] [Support]
hello.I'm using ubuntu 20.04. Can you tell me how to install chia blockchain, create a pilot, etc. with linux codes?

**Comment by Fenex** [comment]
Try to read [wiki](https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide). There is a various topics in the right side.

**Reply by atomadam133** [reply]
my machine is vds so i know the process will be done by typing linux commands
